pyvista/plotting/composite_mapper.py

```diff
@@ -0,0 +1,1032 @@
+
+        Examples
+        --------
+        Disable interpolation before mapping.
+
+        >>> import pyvista as pv
+        >>> dataset = pv.MultiBlock([pv.Cube(), pv.Sphere(center=(0, 0, 1))])
+        >>> pl = pv.Plotter()
+        >>> actor, mapper = pl.add_composite(dataset)
+        >>> mapper.interpolate_before_map = False
+        >>> mapper.interpolate_before_map
+        False
+
+        See :ref:`interpolate_before_mapping_example` for additional
+        explanation regarding this attribute.
+
+        """
+        return bool(self.GetInterpolateScalarsBeforeMapping())
+
+    @interpolate_before_map.setter
+    def interpolate_before_map(self, value: bool):
+        self.SetInterpolateScalarsBeforeMapping(value)
+
+    @property
+    def block_attr(self) -> CompositeAttributes:
+        """Return the block attributes.
+
+        Notes
+        -----
+        ``block_attr`` employs VTK's flat indexing and allows for accessing
+        both the blocks of a composite dataset as well as the entire composite
+        dataset. If there is only one composite dataset, ``A``, which contains
+        datasets ``[b, c]``, the indexing would be ``[A, b, c]``.
+
+        If there are two composite datasets ``[B, C]`` in one composite
+        dataset, ``A``, each of which containing three additional datasets
+        ``[d, e, f]``, and ``[g, h, i]``, respectively, then the head node,
+        ``A``, would be the zero index, followed by the first child, ``B``,
+        followed by all the children of ``B``, ``[d, e, f]``. In data
+        structures, this flat indexing would be known as "Depth-first search"
+        and the entire indexing would be::
+
+           ``[A, B, d, e, f, C, g, h, i]``
+
+        Examples
+        --------
+        Add a sphere and a cube as a multiblock dataset to a plotter and then
+        change the visibility and color of the blocks.
+
+        >>> import pyvista as pv
+        >>> dataset = pv.MultiBlock([pv.Cube(), pv.Sphere(center=(0, 0, 1))])
+        >>> pl = pv.Plotter()
+        >>> actor, mapper = pl.add_composite(dataset)
+        >>> mapper.block_attr[1].color = 'b'
+        >>> mapper.block_attr[1].opacity = 0.1
+        >>> mapper.block_attr[1]
+        Composite Block Addr=... Attributes
+        Visible:   None
+        Opacity:   0.1
+        Color:     Color(name='blue', hex='#0000ffff')
+        Pickable   None
+
+        """
+        return self._attr
+
+    @property
+    def color_missing_with_nan(self) -> bool:
+        """Color missing arrays with the NaN color.
+
+        Examples
+        --------
+        Enable coloring missing values with NaN.
+
+        >>> import pyvista as pv
+        >>> dataset = pv.MultiBlock([pv.Cube(), pv.Sphere(center=(0, 0, 1))])
+        >>> pl = pv.Plotter()
+        >>> actor, mapper = pl.add_composite(dataset)
+        >>> mapper.color_missing_with_nan = True
+        >>> mapper.color_missing_with_nan
```

Comment:
This example's output is not very clear:

![Screenshot from 2022-08-21 00-15-56](https://user-images.githubusercontent.com/17914410/185767831-0532c963-f097-4782-9ecf-9e3608cc41cf.png)

I assume the issue is that there are no "missing values" because there are no scalars chosen at all? Since we already have a full MultiBlock and Plotter set up, can we make this a bit more informative? Or just close the plotter to prevent it from being rendered, perhaps?

Reply:
Agreed. Example improved in 8835c292f03a212cc953200b58f421e82e67c120.